metals/src/main/scala/scala/meta/internal/metals/codeactions/InsertInferredType.scala

```diff
@@ -67,11 +78,47 @@ class InsertInferredType(trees: Trees) extends CodeAction {
       title <- inferTypeTitle(name)
     } yield insertInferTypeAction(title)
 
-    actions.toList
+    def adjustTypeTree(tree: Defn): Option[l.Range] = tree match {
```

Comment:
[nit]
```suggestion
    def defnTreePos(tree: Defn): Option[l.Range] = tree match {
```

or something? `adjustTypeTree` sounds like this method adjust the type tree.

Reply:
Renamed!